src/env.js

```diff
@@ -175,6 +179,10 @@ export class Env {
     get exists() {
 
         const existsProxy = new Proxy(this.source, {
+            /**
+             * @param {any} target 
```

Comment:
Should this be object?

Reply:
it shouldn't really be either `object` or `any`, it should be a generic `T` which extends `NodeJS.ProcessEnv|Deno.Env`, but it didn't matter for my purposes.  we'd have to install some DefinitelyTyped modules to get that to work.

Reply:
You can pass any object into the Env constructor to override the default. Deno.env isn’t the value we use for Deno, it’s Deno.env.toObject(). The only guarantee is that this.source will be an object whose keys are the variables you want access to. You can’t, for instance, use a number of string (which I think “any” allows?)

Reply:
So the more I look at this, the worse it becomes.  I spent awhile with it this morning and discovered that TS is [bad](https://github.com/microsoft/TypeScript/issues/21732) at doing the sort of thing the `@humanwhocodes/env` package does.  In other words, being strict about `this.source` and the types of its keys will be a headache and require much verbosity and typecasting--at minimum.  I also understand that using `object` is kind of fiddly _in general_, and really what TS wants is for any `object` to be strongly typed--otherwise `any` is appropriate, even though it doesn't obviously _make sense_. 

Since we don't know what `this.source` will be, we'd need to go down a rabbit hole of generics and [type guards](https://www.typescriptlang.org/play#example/type-guards) to make all of it compile cleanly.

....

I've removed these docstrings on the `Proxy`s and instead modified the two error helper functions to accept `PropertyKey` values, which is what `Proxy` expects the second parameter (`key`) to be.

You should also know that TS cannot infer that a function throws unless it _itself_ throws.  This means that calling out to a function which _always_ throws is not recommended; TS cannot infer the _calling_ function throws when the call is made.  Anyhow, since _I'm not gonna mess with that stuff_, it doesn't really matter here.

In addition to this, I've made one last change which builds the declarations from `src/env.js` instead of `dist/env.js`, which makes it easier to see TS warnings/errors in the sources if you so chose (otherwise the output `.d.ts` file is roughly identical).

Reply:
(FWIW, I haven't used Deno, and don't know what `Deno.env.toObject()` returns; I was not being literal about the type)